lib/Item/ValueUrlGenerator/TaxonValueUrlGenerator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Item\ValueUrlGenerator;
+
+use Netgen\Layouts\Item\ExtendedValueUrlGeneratorInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+/**
+ * @implements \Netgen\Layouts\Item\ExtendedValueUrlGeneratorInterface<\Sylius\Component\Taxonomy\Model\TaxonInterface>
+ */
+final class TaxonValueUrlGenerator implements ExtendedValueUrlGeneratorInterface
+{
+    public function __construct(private readonly UrlGeneratorInterface $urlGenerator) {}
+
+    public function generateDefaultUrl(object $object): ?string
+    {
+        return $this->urlGenerator->generate(
+            'sylius_shop_partial_taxon_show_by_slug',
+            [
+                'slug' => $object->getSlug(),
+            ],
+        );
+    }
+
+    public function generateAdminUrl(object $object): ?string
+    {
+        return $this->urlGenerator->generate(
+            'sylius_admin_taxon_update',
```

Comment:
Why do you use `update` route? The point is to display the full view of the taxon, both on frontend and on backend.